devices/flash-flexspi/fspi/fspi.c

```diff
@@ -52,31 +52,26 @@ __attribute__((section(".noxip"))) static void flexspi_disable(flexspi_t *fspi,
 __attribute__((section(".noxip"))) static inline void flexspi_swreset(flexspi_t *fspi)
 {
 	*(fspi->base + mcr0) |= 1;
-	while ((*(fspi->base + mcr0) & 1) != 0)
-		;
+	while ((*(fspi->base + mcr0) & 1) != 0) {
+	}
 }
 
 
-__attribute__((section(".noxip"))) void flexspi_lutUpdate(flexspi_t *fspi, u32 index, const u32 *lutTable, size_t count)
+__attribute__((section(".noxip"))) static void flexspi_lutSet(flexspi_t *fspi, u32 index, u32 lutCopy[64], size_t count)
```

Comment:
I think this function may be moved to flexspi_lutUdateEntries as  flexspi_lutUpdate seems redundant.

Reply:
`flexspi_lutUpdate()` is used by the `HyperBus` (flash/ram) implementation, on `imxrt1050-evk` it is the main NOR for this target, so that's why I forked `lutUpdate` into two.